packages/core/src/components/input/input.tsx

```diff
@@ -105,37 +104,18 @@ export class AtomInput {
     return this.inputEl.getInputElement()
   }
 
-  componentDidLoad() {
-    this.inputEl.addEventListener('ionChange', this.handleChange)
-    this.inputEl.addEventListener('ionInput', this.handleInput)
-    this.inputEl.addEventListener('ionBlur', this.handleBlur)
-    this.inputEl.addEventListener('ionFocus', this.handleFocus)
-  }
-
-  disconnectedCallback() {
-    this.inputEl.removeEventListener('ionChange', this.handleChange)
-    this.inputEl.removeEventListener('ionInput', this.handleInput)
-    this.inputEl.removeEventListener('ionBlur', this.handleBlur)
-    this.inputEl.removeEventListener('ionFocus', this.handleFocus)
-  }
-
   private handleChange: IonTypes.IonInput['onIonChange'] = (event) => {
     const value = event.target.value
 
     this.value = value
     this.atomChange.emit(String(value))
   }
 
-  @Watch('value')
-  watchValueHandler(newValue: string) {
-    this.atomChange.emit(String(newValue))
-  }
-
   private handleInput: IonTypes.IonInput['onIonInput'] = (event) => {
     const value = event.target.value
 
     this.value = value
-    this.atomInput.emit(String(value))
+    this.atomChange.emit(String(value))
```

Comment:
question/suggestion: @felipefialho the methods handleChange and handleInputs are equal. What about use only handleChange on `onIonChange` and `onIonInput`?

Reply:
@igorwessel @felipefialho reverted https://github.com/juntossomosmais/atomium/pull/330/commits/3f65743f0aecb29e7a4c25023e46367553d1e8f8